### DISCUSSION TITLE
Does mpdf plan to support PDF/A-3?

### DISCUSSION THREAD

**Discussion by lines-dev** [mpdf/mpdf#1568] [Q&A]
mpdf can attach files “PDF/A-3” way, but the the generated pdf file's standard is PDF/A-1B.

Does mpdf plan to fully support PDF/A-3?

**Comment by finwe** [comment]
Not at the moment.

**Reply by kevinpapst** [reply]
Edit: according to https://github.com/mpdf/mpdf/discussions/2071
this might be the solution
https://mpdf.github.io/reference/mpdf-variables/pdfaversion.html
